web-client/src/views/FileDocument/StateDrivenFileInput.jsx

```diff
@@ -9,6 +9,7 @@ export const StateDrivenFileInput = connect(
   {
     ariaDescribedBy: props.ariaDescribedBy,
     constants: state.constants,
+    file: props.file,
```

Comment:
Have a look at the linting catch below. `file` is already defined. We might want to break out that function or use a more descriptive name here for `file`.